internal/db/migrations/010_rename_provider_detail.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE aws_reservation_details RENAME provider TO provider_fk;
```

Comment:
Why are we renaming to `_fk`? Sounds like a weird col name, I'd expect constraint having this name

Reply:
There was a conflicts for the joined tables, otherwise we would need to drop "*" from the SQL replacing it with 13 columns. I thought I would rename instead of this.

Reply:
ok, make sense, I'd still love less confusing name, but it's fine, not a big deal at all, just curious :) Thanks for the explanation :+1: